Makefile

```
@@ -187,4 +200,4 @@ release-pypi:
 
 ci: check coverage
 
-.PHONY: check clean pylint pep8 install tag archive local
+.PHONY: check check-deps show-deps clean pylint pep8 install tag archive local
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)